README.md

```diff
@@ -60,9 +30,13 @@ The following Helm values are available:
 - `selector`: determines which `TrafficSplit` instances to consider for
   failover. It defaults to `failover.linkerd.io/controlled-by={{.Release.Name}}`
   (the value refers to the release name used in `helm install`).
+- `logLevel`, `logFormat`: for configuring the operators logging.
 
 ## Installation
 
+The SMI extension and the operator are to be installed in the local cluster
+(there where the clients consuming the service are located).
```

Comment:
I think "there" makes this a bit more confusing. Do you think the sentence would still make sense if we remove it? 

```
The SMI extension and the operator are to be installed in the local cluster (where the clients consuming the service are located).
```

Reply:
:+1: